### PR TITLE
MAP-347

### PR DESCRIPTION
**Pull request by danbenton-mojdt**
### What changed

Added option to book extradition moves when current location is extradition_capable.
List of available destinations is restricted to extradition_capable custody suites
Added extradition details page to move booking form
Add extradition details section to move details for extradition moves

DEPENDENCY UPDATES
Govuk frontend has moved to v5.x, changing some behaviours and styling across the service
date-fns has moved to v3, affecting many functions in the app, especially ordering of lists by time

### Why did it change

Supplier is now able to see details of extradition flights when provided by the user during booking

### Issue tracking

- MAP-347

## Screenshots
image

image

image

#### Automated testing

- [x] Added unit tests to cover changes
- [x] Added end-to-end tests to cover any journey changes

#### Manual testing

Has been tested in the following browsers:

- [x] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Internet Explorer

### Environment variables

- [ ] Documented in the [README](ministryofjustice/hmpps-book-secure-move-frontend/blob/main/README.md)
- [x] Added to [continous integration](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-book-secure-move-frontend)
- [x] Added to Helm config

### Other considerations

Commit messages with a `fix` or `feat` type are automatically used to generate the [changelog](ministryofjustice/hmpps-book-secure-move-frontend/blob/main/CHANGELOG.md) and
[GitHub release notes](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/releases) during the release task. Please make sure they will read well on their own in a
summary of changes and that the commit body gives a more detailed description of those changes if necessary.

- [x] No new [Personally Identifiable Information (PII)](https://support.google.com/analytics/answer/6366371?hl=en) is being sent via analytics
- [ ] Update [README](ministryofjustice/hmpps-book-secure-move-frontend/blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: --fix/_update-notifier-last-checked
Comment (jimbali) [inline --fix/_update-notifier-last-checked:1]: Did something go weird here?

**Review by jimbali — COMMENTED** [review]
Just a few minor comments. My only bugbear is that some of the new files are JS rather than TS, but I guess it might be tricky with some files like the fields for example?

File: --fix/_update-notifier-last-checked
Comment (danbenton-mojdt) [reply --fix/_update-notifier-last-checked:1]: Apparently so, removed

File: package.json [hunk 1]
```diff
@@ -363,7 +364,7 @@
     "engine-strict": true
   },
   "lint-staged": {
-    "*.js": "eslint --cache",
+    "*.js": "eslint --cache --fix",
```
[the remark this answers is not in the document]
Comment (danbenton-mojdt) [reply package.json:367]: Correct, removed

File: common/formatters/index.ts [hunk 2]
```diff
@@ -5,14 +5,24 @@ const { oxfordJoin, nonOxfordJoin } = require('../../config/nunjucks/filters')
...
+  const parsedTime = parsers.time(value)
+  return parsedTime
 }
 
 export function time (value: Date | string, format = 'HH:mm') {
   const parsedDate = parsers.date(value)
   return parsedDate ? formatDate(parsedDate, format) : value
 }
 
+
```
[the remark this answers is not in the document]
Comment (danbenton-mojdt) [reply common/formatters/index.ts:25]: Removed

File: app/move/app/new/controllers/extradition-details.js
[the remark this answers is not in the document]
Comment (danbenton-mojdt) [reply app/move/app/new/controllers/extradition-details.js:1]: I considered this, the ones that are js files are the same pattern as other files in the same dir doing the same thing - eg fields as mentioned above.  In the those cases I think we should convert them all to ts but this PR was already snowballing with the update to govuk frontend v5 and date-fns v3

File: --fix/_update-notifier-last-checked
Comment (jimbali) [reply --fix/_update-notifier-last-checked:1]: still seems to be there...

Comment (jimbali) [reply --fix/_update-notifier-last-checked:1]: ah it was just GitHub being "helpful"

**Review by jimbali — APPROVED** [review]

**Pull request merged**